### ISSUE TITLE
meteor update --release 1.2-rc.5 results in error

### ISSUE THREAD

**Issue by zimme** [meteor/meteor#5041]
```sh
➜  devshop  meteor update --release 1.2-rc.5

/Users/zimme/.meteor/packages/meteor-tool/.1.1.7-rc.5.dprmdu++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/tools/cordova/index.js:1
(function (exports, require, module, __filename, __dirname) { import _ from 'u
                                                              ^^^^^^
SyntaxError: Unexpected reserved word
    at Module._compile (module.js:439:25)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (/Users/zimme/.meteor/packages/meteor-tool/.1.1.7-rc.5.dprmdu++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/tools/cli/commands.js:5:16)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (/tools/cli/main.js:285:1)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (/tools/index.js:5:1)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Function.Module.runMain (module.js:497:10)
    at startup (node.js:119:16)
    at node.js:935:3
```

cc @dgreensp

**Comment by bySabi** [comment]
same here!

**Comment by rclai** [comment]
I don't think their RCs are ready for testing yet according to the main 1.2 release thread.

**Comment by bySabi** [comment]
now working with `rc-4` without problems.
What is the point of release a `release` that not can be tested?

**Comment by sclausen** [comment]
@bySabi +1

**Comment by zimme** [comment]
@rclai. That might be true, but once they put a RC release out there people are gonna start to use it.

I'm just trying to get an early start on making sure my packages works as expected with the new version.

**Comment by dgreensp** [comment]
We have to build and publish an RC in order to do our full testing and QA (e.g. testing `meteor update`).  It may not work, but you are always welcome try it.  I appreciate the enthusiasm.

We're going to have an RC ready to announce very soon (see #4943).

**Comment by zimme** [comment]
:+1:

**Comment by bySabi** [comment]
good to hear:  `very soon` :-)

by the way @dgreensp  in my system, OSX, `meteor update` not work with `rc-4` but I can create projects with `meteor --release 1.2-rc.4 create 'projectName'` and is ok.

With rc-5 I can´t do `meteor --release 1.2-rc.5 create 'projectName'` fail as in post @zimme

**Comment by dgreensp** [comment]
Fixed in a2a8192c357df943ecd0e6da0cceb91ed89854bb.
